integration-tests/tests/test_identity_transformation.py

```diff
@@ -0,0 +1,122 @@
+import shutil
+from pathlib import Path
+from tempfile import NamedTemporaryFile
+from typing import IO
+
+import pytest
+from tests.utils.config import (
+    BaseConfig,
+    DatasetLogs,
+)
+from tests.utils.utils import (
+    diff_equal,
+    run_and_assert,
+)
+
+pytestmark = pytest.mark.binaries
```

Comment:
I am not sure if we need to mark the test with both `binaries` here and `clp/clp-s` in front of each test. At least in this PR, I don't see the advantage, any insights? @kirkrodrigues

Reply:
As discussed with Kirk, we can keep the markers. They are functional if we want to selectively run tests.
`binaries` will be useful once we add other tests that test the package lifecyle (start/stop)